api/ws/rpc/wsRPC.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+var MasterWAMPServer = require('wamp-socket-cluster/MasterWAMPServer');
+var scClient = require('socketcluster-client');
+var WAMPClient = require('wamp-socket-cluster/WAMPClient');
+var Q = require('q');
+var _ = require('lodash');
+var constants = require('../../../helpers/constants');
+
+var wsRPC = {
+
+	clientsConnectionsMap: {},
+	scClient: scClient,
+	wampClient: new WAMPClient(),
+	wsServer: null,
+
+	/**
+	 * @param {MasterWAMPServer} wsServer
+	 */
+	setServer: function (wsServer) {
```

Comment:
Yep, it looks good - but should be the reason, maybe move those exposed properties to private variables?